makefile

```diff
@@ -1,9 +1,11 @@
 CC = gcc
 AL_LIBS = $(shell allegro-config --libs)
-CFLAGS ?= -O2 -g -Wall
```

Comment:
I'd rather this stay the default. Anyone wanting to ship without symbols can strip the binary before release.

Reply:
OK, as you want